lib/tasks/gettext.rake

```diff
@@ -18,11 +18,13 @@ namespace :gettext do
   end
 
   desc 'Generate POT file'
-  task :pot do
+  task :pot, [:filename] do |_, args|
     begin
-      result = GettextSetup::Pot.generate_new_pot
+      filename = args.filename || ENV['filename']
```

Comment:
Do we want to read this generic environment variable? I think standardizing on just one way to call the task is fine.

Reply:
@tphoney , I wasn't clear on what we needed `ENV['filename']` for or not, so I just left it. Perhaps we can pitch it?